techniques/system/common/1.0/update.st

```diff
@@ -264,7 +264,7 @@ bundle agent update_action
         classes           => success("config", "no_update", "config_ok");
 
     # We copy ncf on non root server if ncf hash file changed, or if  promises have been updated
-    !root_server.(rudder_promises_generated_repaired|rudder_ncf_hash_update_repaired)::
```

Comment:
This class no longer exists in the 3.2 branch, and got here by a merge up from 3.x branches